packages/fast-components-react-base/src/horizontal-overflow/horizontal-overflow.spec.tsx

```diff
@@ -192,13 +192,29 @@ describe("horizontal overflow", (): void => {
 
         expect(renderedWithImages.find("img").length).toBe(6);
     });
-    test("should not set `itemHeight` if no children have been passed", () => {
+    test("should not set `itemHeight` if no children have been passed or fixedHeight set", () => {
         const rendered: any = mount(
             <HorizontalOverflow managedClasses={managedClasses} />
         );
 
         expect(rendered.state("itemsHeight")).toBe(null);
     });
+    test("should set `itemHeight` to the fixedHeight if no children have been passed but a fixedHeight has been set", () => {
+        const rendered: any = mount(
+            <HorizontalOverflow managedClasses={managedClasses} fixedHeight={100} />
+        );
+
+        expect(rendered.state("itemsHeight")).toBe(100);
+    });
+    test("should set `itemHeight` to the fixedHeight if children have been passed but a fixedHeight has been set", () => {
```

Comment:
NIT: should set `itemHeight` to the fixedHeight if children have been passed **and** a fixedHeight has been set